tests/components/axis/test_init.py

```diff
@@ -75,3 +75,41 @@
         axis.CONF_EVENTS: True,
         axis.CONF_TRIGGER_TIME: axis.DEFAULT_TRIGGER_TIME,
     }
+
+
+async def test_migrate_entry(hass):
+    """Test successful migration of entry data."""
+    legacy_config = {
+        axis.CONF_DEVICE: {
+            axis.CONF_HOST: "1.2.3.4",
+            axis.CONF_USERNAME: "username",
+            axis.CONF_PASSWORD: "password",
+            axis.CONF_PORT: 80,
+        },
+        axis.CONF_MAC: "mac",
+        axis.device.CONF_MODEL: "model",
+        axis.device.CONF_NAME: "name",
+    }
+    entry = MockConfigEntry(domain=axis.DOMAIN, data=legacy_config)
+
+    assert entry.data == legacy_config
+    assert entry.version == 1
+
+    await axis.async_migrate_entry(hass, entry)
```

Comment:
Use `entry.async_migrate` instead.

https://github.com/home-assistant/home-assistant/blob/d5486f883d8063a38dd99e22047c32877c14b3ff/homeassistant/config_entries.py#L332

Reply:
Will fix in separate PR 👍